.github/workflows/integration-tests.yml

```diff
@@ -23,7 +23,16 @@ jobs:
           sudo apt update
           sudo apt install -y ansible
 
-      - name: Run test playbook
+      - name: Install dependencies
+        run: |
+          set -euo pipefail
+          for ff in meta/collection-requirements.yml tests/collection-requirements.yml; do
+            if [ -f "$ff" ]; then
```

Comment:
Feels a little brittle/unnecessarily complicated -- we *know* that the first exists, and the second doesn't. But not a strong opinion.

Reply:
> Feels a little brittle/unnecessarily complicated -- we _know_ that the first exists, and the second doesn't. But not a strong opinion.

I believe it make this less brittle/more robust - https://linux-system-roles.github.io/documentation/role-requirements.html
```
Collections required at test-time will be specified in tests/collection-requirements.yml. Users will typically not need to know about this, but developers can use this with tox-lsr, the CI system, etc.
```
so if we do add test dependencies in the future, CI will "just work", and the developer doesn't have to know to change the github workflow file.

And - this is how all of our other CI works - https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/test_scripts/runqemu.py#L949 - and - https://gitlab.com/testing-farm/gluetool-modules/-/blob/main/gluetool_modules_framework/testing/test_scheduler_system_roles.py#L194 - so it makes cockpit integration testing consistent with other system roles integration testing.